src/main/resources/static/templates/JWTVulnerability/keys/private_key.pem

```diff
@@ -1,28 +1,32 @@
+Bag Attributes
```

Comment:
Is this key being used anywhere ?

Reply:
Yes they are used in JWT Vulnerability. Actually not directly but in case we want to test ZAP fuzzer we use these.